test/test_transaction.py

```diff
@@ -23,26 +27,34 @@
 class TestTransaction:  # pylint: disable=too-many-public-methods
     """Unittest for the pymodbus.transaction module."""
 
-    client = None
-    decoder = None
-    _tcp = None
-    _tls = None
-    _rtu = None
-    _ascii = None
-    _manager = None
-    _tm = None
+    client: ModbusBaseSyncClient
+    decoder: ServerDecoder
+    _tcp: ModbusSocketFramer
+    _tls: ModbusTlsFramer
+    _rtu: ModbusRtuFramer
+    _ascii: ModbusAsciiFramer
+    _manager: SyncModbusTransactionManager
```

Comment:
So...if we remove these mypy doesn't care but pylint then complains, when they are set to `None` then mypy does complain, best solution I've come up with so far is to just set the types here like this as this plays well with both mypy and pylint.

Reply:
Yeah, agreed.  I forgot about `pylint` since I almost never run it :S